eras/conway/test-suite/cardano-ledger-conway-test.cabal

```
@@ -38,17 +38,17 @@ library
         base >=4.14 && <4.19,
         cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} ^>=1.3,
         cardano-ledger-alonzo-test,
-        cardano-ledger-babbage ^>=1.3,
+        cardano-ledger-babbage ^>=1.4,
```

The point about this range is that nothing that changed in the dependency is needed in the package, so there is no need to artificially make it incompatible with an older dependency
```suggestion
        cardano-ledger-babbage >=1.3 && <1.5,
```

Also every change to cabal requires a patch version update, so `1.2.0.1`